src/components/fields/derived.js

```diff
@@ -105,14 +105,45 @@ export const AxesRange = connectToContainer(UnconnectedNumeric, {
   },
 });
 
-class NumericFraction extends UnconnectedNumeric {}
-NumericFraction.propTypes = UnconnectedNumeric.propTypes;
-NumericFraction.defaultProps = {
+class UnconnectedNumericFraction extends UnconnectedNumeric {}
+UnconnectedNumericFraction.propTypes = UnconnectedNumeric.propTypes;
+UnconnectedNumericFraction.defaultProps = {
   units: '%',
+  showSlider: true,
 };
 
+const numericFractionModifyPlotProps = (props, context, plotProps) => {
+  const {attrMeta, fullValue, updatePlot} = plotProps;
+  const min = attrMeta.min || 0;
+  const max = attrMeta.max || 1;
+  if (isNumeric(fullValue)) {
+    plotProps.fullValue = Math.round(100 * (fullValue - min) / (max - min));
+  }
+
+  plotProps.updatePlot = v => {
+    if (isNumeric(v)) {
+      updatePlot(v / 100 * (max - min) + min);
+    } else {
+      updatePlot(v);
+    }
+  };
+  plotProps.max = 100;
+  plotProps.min = 0;
+};
+
+export const NumericFraction = connectToContainer(UnconnectedNumericFraction, {
+  modifyPlotProps: numericFractionModifyPlotProps,
+});
+
```

Comment:
ok, so this is my general control for when I want a slider instead of a standard numeric.

Reply:
only if you want it to be expressed as a percentage! if your min is 2 and your max is 4 then 50% = 3 etc... if you want a slider from 2 to 3 you should use `<Numeric showSlider ... >`